pypdf/_xobj_image_helpers.py

```diff
@@ -112,30 +112,30 @@ def _get_imagemode(
         "2bit": "2bits",  # 2 bits images
         "4bit": "4bits",  # 4 bits
     }
-    mode: mode_str_type = (
+    mode_: mode_str_type = (
         mode_map.get(color_space)  # type: ignore
         or list(mode_map.values())[color_components]
         or prev_mode
     )
-    return mode, mode == "CMYK"
+    return mode_, mode_ == "CMYK"
 
 
 def bits2byte(data: bytes, size: Tuple[int, int], bits: int) -> bytes:
     mask = (1 << bits) - 1
-    nbuff = bytearray(size[0] * size[1])
-    by = 0
+    buff = bytearray(size[0] * size[1])
+    byt = 0
```

Comment:
I would probably avoid abbreviations altogether and use the names `byte_buffer` and `data_index` here.